src/index.js

```diff
@@ -1,21 +1,27 @@
-import _ from 'lodash';
+import Task from './modules/tasks.js';
 import './style.css';
-import Icon from './icon.png';
+import {
+  renderTasks, addButton, addInput, clearAllButton,
+} from './modules/todo.js';
 
-function component() {
-  const element = document.createElement('div');
+renderTasks(Task);
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+addButton.addEventListener('click', () => {
+  const description = addInput.value;
+  if (description) {
+    Task.addTask(description);
+    addInput.value = '';
+    renderTasks(Task);
```

Comment:
According to the project requirements, by default, your app should contain a hardcoded array with some simple to-do tasks containing three keys. Then the tasks are used to populate a HTML list item. Please update your application to include this feature.